lib/ui_components/cell.rb

```diff
@@ -1,5 +1,14 @@
+require_relative '../../app/helpers/ui_components/view_helper'
```

Comment:
The only thing it does is providing the interface to the outside world. Using this *from the inside* as well gives us the benefit of not having to change all the places where a component is used within another one, when the implementation changes and the interface stays the same.